kytos.json

```diff
@@ -0,0 +1,11 @@
+{
+  "author": "kytos",
+  "username": "kytos",
+  "name": "of_multi_table",
+  "description": "This NApp implements Oplenflow multi tables",
+  "version": "1.0",
+  "napp_dependencies": [],
+  "license": "",
```

Comment:
Can you also update this as `"MIT"` license and include the `LICENSE` file too? You can copy as it is from any of the NApps, but then in the copyright line something like `Copyright (c) 2023 Kytos-ng Team`

Reply:
Added